changelog.md

```diff
@@ -0,0 +1,40 @@
+# Changelog
+All notable changes to this project will be documented in this file.  
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [1.0](https://github.com/MariaMaximova/convert_temp/tree/v1.0) - 2022-August-06
```

Comment:
@MariaMaximova, please keep in mind that all versions go from bottom to top. Please reverse the versions so the most recent changes are on top of the list

Reply:
[List](https://github.com/MariaMaximova/convert_temp/pull/9/commits/1ab8b94e86226a891571c2a8960d6bb6b781e07d#diff-3bd14d078188074c410028847113ceae68865d0ad5b844a27183ef87fbe2fcc3) of the versions was reversed